packages/babel-plugin-transform-es2015-parameters/test/fixtures/parameters/regression-5787/expected.js

```diff
@@ -0,0 +1,8 @@
+function f(a) {
+  var _ref, _ref2, _ref3;
+
+  var b = (_ref = (arguments.length <= 1 ? 0 : arguments.length - 1) - 3 + 1, arguments.length <= _ref || _ref < 1 ? undefined : arguments[_ref]);
```

Comment:
This first ternary (`ref = (arguments.length <= 1 ? 0 : argument.length - 1)`) shouldn't be necessary now.